From 947be0f9f2e2060435650527f3eea8d60489a80f Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Thu, 14 Apr 2016 08:28:51 +0300
Subject: [PATCH 356/538] defconfig: Enable nvme support in v8_lsp_defconfig

Needed for some PCIe performance measurement over NVME devices.

Change-Id: I949bcff421ece5e42be8b4b0726564a8a2db744b
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29138
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 74db95c..7ffc75e 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -79,6 +79,7 @@ CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_COUNT=1
 CONFIG_BLK_DEV_RAM_SIZE=48000
 CONFIG_VIRTIO_BLK=y
+CONFIG_BLK_DEV_NVME=y
 # CONFIG_SCSI_PROC_FS is not set
 CONFIG_BLK_DEV_SD=y
 # CONFIG_SCSI_LOWLEVEL is not set
-- 
1.9.1

